src/index.html

```diff
@@ -5,8 +5,323 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin
+      >
```

Comment:
```suggestion
    <link
      rel="preconnect"
      href="https://fonts.gstatic.com"
      crossorigin
    >
```
Должны быть на одной линии